### DISCUSSION TITLE
gtag cross domain

### DISCUSSION THREAD

**Discussion by Visualapproch** [AmauriC/tarteaucitron.js#936] [General]
Bonjour,

Nous avons un problème de cross domain qui ne fonctionne pas avec tarteaucitron:
Nous avons remplacé ce script qui fonctionne:

`window.dataLayer = window.dataLayer || [];
function gtag(){dataLayer.push(arguments);}
gtagBonjour,

Nous avons un problème de cross domain qui ne fonctionne pas avec tarteaucitron:
Nous avons remplacé ce script qui fonctionne:

`window.dataLayer = window.dataLayer || [];
function gtag(){dataLayer.push(arguments);}
gtag('js', new Date());
gtag('config', 'UA-XXX-1');
gtag('set', 'linker', {
 'domains': ['domaine1.com', 'domaine2.com'],
 'decorate_forms': true
});`

Par 
`tarteaucitron.user.gtagUa = 'UA-546347-1';
 tarteaucitron.user.gtagCrossdomain = ['domaine1.com', 'domaine2.com'];
tarteaucitron.user.gtagMore = function () { gtag('config', 'UA-XXX-1', {'decorate_forms': true}) };	
(tarteaucitron.job = tarteaucitron.job || []).push('gtag');`

Qui ne fonctionne pas quand l'utilisateur de domaine1.com arrive sur domaine2.com

Avez-vous une idée pour résoudre ce problème?
Merci('js', new Date());
gtag('config', 'UA-XXX-1');
gtag('set', 'linker', {
 'domains': ['domaine1.com', 'domaine2.com'],
 'decorate_forms': true
});`

Par 
`tarteaucitron.user.gtagUa = 'UA-546347-1';
 tarteaucitron.user.gtagCrossdomain = ['domaine1.com', 'domaine2.com'];
tarteaucitron.user.gtagMore = function () { gtag('config', 'UA-XXX-1', {'decorate_forms': true}) };	
(tarteaucitron.job = tarteaucitron.job || []).push('gtag');`

Qui ne fonctionne pas quand l'utilisateur de domaine1.com arrive sur domaine2.com

Avez-vous une idée pour résoudre ce problème?
Merci

**Comment by AmauriC** [comment]
Bonjour,

Qu'est ce qui ne fonctionne pas? Tarteaucitron charge GA sur le 2ème site mais il n'y a pas de remonté sur la console Google?
